src/OSPSuite.UI/Controls/UxMasterDetailGridView.cs

```diff
@@ -0,0 +1,13 @@
+using DevExpress.XtraGrid;
+
+namespace OSPSuite.UI.Controls
+{
+   public class UxMasterDetailGridView : UxGridView
+   {
+      protected override string ViewName => "UxMasterDetailGridView";
```

Comment:
What I mean is, I think by changing that ViewName, we are introducing some new behavior in UxGridView that breaks something else.